### ISSUE TITLE
[Feature Request]: Option to stop "Play background music" 

### ISSUE THREAD

**Issue by gsxgmpkt9020** [ollm/OpenComic#298]
### Preflight Checklist

- [X] I have searched the [issue tracker](https://www.github.com/ollm/OpenComic/issues) for a feature request that matches the one I want to file, without success.

### Describe your suggested feature

It is quite annoying that "Play background music" is a default action. Users should have right to know that opencomic has this feature opened. Most people don't know this and will think this is a bug.

I usually need to open a cbz file in a folder which has all kinds of files including audio media.

Please give us a option to turn this off.

I found this function in https://github.com/ollm/OpenComic/issues/261.

I think "Play music in background in the folder" should not be turned on when most people don't know.

**Comment by ollm** [comment]
The option can be disabled with the icon that appears when music is present, it is a global option (like the magnifying glass) so it will apply to all folders.

![Captura desde 2024-11-29 17-55-00](https://github.com/user-attachments/assets/b67d9c90-e29a-4fe8-8977-1295c45f8836)

If you think that's better, I can change the default option for new installations to false.
https://github.com/ollm/OpenComic/blob/342c9feea652fab1fa4444e07a7dc32afaa859e3/scripts/storage.js#L151-L154

**Comment by ollm** [comment]
You are right, it is better to activate it expressly if you want to use it (Mainly for Webtoons https://github.com/ollm/OpenComic/issues/149), I have changed it to `false` and the change will be in the next version b6346a71d77b735b3383a5ffd475d807134f53c3
